recipes/gz-sim-yarp-plugins/meta.yaml

```diff
@@ -0,0 +1,93 @@
+{% set cxx_name = "libgz-sim-yarp-plugins" %}
+{% set plugin_name = "gz-sim-yarp-plugins" %}
+{% set name = "gz-sim-yarp-plugins-split" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/robotology/gz-sim-yarp-plugins/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 203931dc039170ae388146004356daf1619a2af21fd56c2edb684ec4d39db89b
+
+build:
+  number: 0
+
+outputs:
+  - name: {{ cxx_name }}
+    script: build_cxx.sh  # [unix]
+    script: bld_cxx.bat  # [win]
+    build:
```

Comment:
This probably should have a run_exports?

Reply:
Yes, it will have it eventually. However, at the moment the package does not install any header file, so it can't be actually consumed as a C++ library by downstream users (it can only be used at runtime by loading it via `dlopen`). Furthermore, we still need to define an ABI stability policy, so we did not add any `run_exports`. As soon as we do a release which install headers and define an ABI stability policy, we will add `run_exports` . See also the original post: https://github.com/conda-forge/staged-recipes/pull/25910#issue-2215228329 .